torchaudio/csrc/CMakeLists.txt

```diff
@@ -5,11 +5,11 @@ get_property(TORCHAUDIO_THIRD_PARTIES GLOBAL PROPERTY TORCHAUDIO_THIRD_PARTIES)
 ################################################################################
 set(
   LIBTORCHAUDIO_SOURCES
+  sox/types.cpp
   sox/io.cpp
   sox/utils.cpp
   sox/effects.cpp
   sox/effects_chain.cpp
-  sox/types.cpp
```

Comment:
Is this your IDE or some linter changing the order? Unless there is a strong reason, we prefer to keep diff simpler. (If a linter is suggesting this, it's fine